### ISSUE TITLE
Access The Data: Update alt text for Access The Data image to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3870]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text for the `Access The Data` image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/access-the-data.md`:

**From:**
```
alt: 'description of the card image'
```

**To:**
```
alt: 'Access The Data'
```

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- Access The Data Page: https://www.hackforla.org/projects/access-the-data

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/access-the-data.md`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/_projects/access-the-data.md
- This issue is part of epic #3866 

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by mademarc** [comment]
Availability: 4/17/2023
ETA: 4/20/2023

**Comment by amath95** [comment]
Availability : 04/02
ETA : 04/09
